docs/source/playground/index.ejs

```diff
@@ -442,14 +442,22 @@ meta_description: Label Studio interactive demo and playground for data labeling
     return "https://label-studio-playground.netlify.app/";
   }
 
+  function normalizeNewlines(text) {
+    return text.replace(/(\r\n|\n|\r)/gm, "\n");
```

Comment:
`\n` is redundant here I guess? it's not appearing in next options in this union, so should be safe to remove
```suggestion
    return text.replace(/(\r\n|\r)/gm, "\n");
```

Reply:
that is true, yep let me remove it from the union options.